Hymns/Infrastructure/Analytics/AnalyticsLogger.swift

```diff
@@ -24,6 +24,10 @@ class AnalyticsLogger {
         ])
     }
 
+    func logDisplayMusicPDF(url: URL) {
+        Analytics.logEvent(DisplayMusicPDF.name, parameters: [ DisplayMusicPDF.Params.display_musicPDF.rawValue: url])
```

Comment:
Did you check to see how this appears on Firebase's DebugView? I'm not sure it knows how to log a URL object